plugins/explain.py

```diff
@@ -82,7 +82,19 @@ class Explain(BotPlugin):
                      'newcomer Pull Request\n- review at least a difficulty/'
                      'newcomer Pull Request\n- merge a difficulty/low Pull '
                      'Request\n- review at least a difficulty/low or higher '
-                     'Pull Request'
+                     'Pull Request',
+        'do not ask to ask': 'This seems a lazy way of asking your question '
+                             'Don\'t take it personally. It\'s just that '
```

Comment:
use double quotes around the whole string instead of escaping the single quotes. That way you can keep the single quotes in the string more readable. (this comment applies to all of the instances)